api/wasm.go

```diff
@@ -327,6 +327,18 @@ type Memory interface {
 	Write(ctx context.Context, offset uint32, v []byte) bool
 }
 
+// EncodeExternref encodes the input as a ValueTypeExternref.
+// See DecodeExternref
+func EncodeExternref(input uintptr) uint64 {
```

Comment:
while this is trivial, it formalizes the documentation on ValueTypeExternref where uintptr is how we represent externtype